solidity/contracts/BancorNetwork.sol

```diff
@@ -88,7 +88,7 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient {
         public

```

Comment:
@yudilevi : One thing to keep in mind is that these revert messages may be relied upon by our users (dapps) in order to determine state/result. So any change that we make in any of them must be posted as an API-breaking change.